### ISSUE TITLE
Are you able to train the datasets and register the images?

### ISSUE THREAD

**Issue by ku294714** [evarcurto/myDiffuseMorph#2]

**Comment by evarcurto** [comment]
Hi @ku294714,

I was able to train the model and register images only with my own dataset (nothing related to faces).

**Comment by ku294714** [comment]
Hello @evarcurto , Thank you for the quick reply, did you make any changes in the architecture of the model/hyperparameters. Also is the model converging for your dataset in 40 epochs or you have to run it for may epochs?

**Comment by evarcurto** [comment]
Yes, I had to test with different hyperparameters until I found the most suitable. I trained my dataset during 200 epochs with le=5.00e-05 and loss lambda=0.5.

**Comment by ku294714** [comment]
Perfect. Thank you for your reply..
